doc/level1.rst

```diff
@@ -6,22 +6,42 @@
 Autocrypt Level 1: Enabling encryption, avoiding annoyances
 ===========================================================
 
-Autocrypt makes it easy for people to encrypt email.  This document
-describes the basic capabilities required for a mail app to be
-Autocrypt-capable at Level 1.
-
-The design of Level 1 is driven by usability concerns and by the
-realities of incremental deployment. A user may mix both
-Autocrypt-enabled MUAs and traditional MUAs, and we'd
```

Comment:
this update seems to have lost this "may mix both Autocrypt-enabled MUAs and traditional MUAs" idea.  do we not need to express it any more?

Reply:
added it back with thenew commit.